isilon_create_directories.sh

```diff
@@ -173,6 +173,7 @@ case "$DIST" in
             "/user/spark#751#spark#spark" \
             "/user/spark/applicationHistory#1777#spark#spark" \
             "/user/sqoop2#775#sqoop2#sqoop" \
+            "/user/yarn/mapreduce/mr-framework#755#yarn#hadoop" \
```

Comment:
I think this should be just `/user/yarn`. Any subdirectories should be taken care of by that change since yarn can create directories below this.

In issue #68 I mentioned that but didn't give a complete description.